### PR TITLE
Add support for ESXi 7 in Cobbler 3

### PR DESCRIPTION
**Pull request by SchoolGuy**
Supersedes <https://github.com/cobbler/cobbler/pull/2389>

Authored by: Jason Hong-Turney (GH: @Jhongturney)

### CODE REVIEW

File: config/cobbler/distro_signatures.json [hunk 1]
```diff
@@ -673,38 +673,7 @@
           ]
         }
       },
-      "fedora32": {
```
Comment (dansou901) [inline config/cobbler/distro_signatures.json:676]: Why remove fedora32?

File: config/cobbler/distro_signatures.json [hunk 2]
```diff
@@ -1336,32 +1305,6 @@
         "kernel_options": "",
         "kernel_options_post": "",
         "boot_files": []
-      },
-      "focal": {
```
Comment (dansou901) [inline config/cobbler/distro_signatures.json:1340]: Why remove ubuntu focal?

**Review by dansou901 — CHANGES_REQUESTED** [review]
Why remove fedora32 and ubuntu focal in distro_signatures.json? This should be changed.

File: config/cobbler/distro_signatures.json (hunk 1 above)
Comment (SchoolGuy) [reply config/cobbler/distro_signatures.json]: Fixed. Thanks for the review. Copy Pasta Error...

File: config/cobbler/distro_signatures.json (hunk 2 above)
Comment (SchoolGuy) [reply config/cobbler/distro_signatures.json]: Fixed. Thanks for the review. Copy Pasta Error...

**Review by dansou901 — APPROVED** [review]
LGTM now

**Comment by SchoolGuy** [comment]
Tests green, no objections. Let's merge this, so it is in 3.2.0.

**Pull request merged**